cmake/Platform/Linux/PAL_linux.cmake

```diff
@@ -53,16 +53,8 @@ ly_set(LY_PYTHON_CMD ${CMAKE_CURRENT_SOURCE_DIR}/python/python.sh)
 set(PAL_TRAIT_LINUX_WINDOW_MANAGER "xcb" CACHE STRING "Sets the Window Manager type to use when configuring Linux")  
 set_property(CACHE PAL_TRAIT_LINUX_WINDOW_MANAGER PROPERTY STRINGS xcb wayland)
 
-# Use system default OpenSSL library instead of maintaining an O3DE version for Linux
-include(${CMAKE_CURRENT_LIST_DIR}/OpenSSL_linux.cmake)
-
 # Use system default libunwind  instead of maintaining an O3DE version for Linux
```

Comment:
While you are in here, can you remove the extra space between the words of "libunwind" and "instead"
```suggestion
# Use system default libunwind instead of maintaining an O3DE version for Linux
```